learn/guides/ballerina-shell.md

```diff
@@ -20,7 +20,7 @@ This will particularly be useful if you are looking to try out the language and
 
 ## Setting up the Prerequisites
 
-Ballerina Shell is a part of the Ballerina distribution. Therefore, if you already installed Ballerina, you are all set. If not, for information on installing the latest version of Ballerina, see [downloads](https://ballerina.io/downloads/).
+Ballerina Shell is a part of the Ballerina distribution. Therefore, if you have already installed Ballerina, you are all set. If not, for information on installing the latest version of Ballerina, see [downloads](https://ballerina.io/downloads/).
```

Comment:
```suggestion
Ballerina Shell is a part of the Ballerina distribution. Therefore, if you have already installed Ballerina, you are all set. If not, for information on installing the latest version of Ballerina, see [downloads](https://ballerina.io/downloads/).
```
This is correct. It should either be "If you already have Ballerina installed" or "If you already installed Ballerina".

Reply:
<img width="1303" alt="image" src="https://user-images.githubusercontent.com/16300038/151326465-46348343-1bb7-4a6a-bdd5-77b7c26a34f3.png">

Quoting from grammerly, can u explain why it suggests the otherway?

Reply:
Not sure what exactly is the reason as both are in the present tense when we say `if you already installed Ballerina, you are all set.`

@dakshitha - Any idea?

Reply:
I am not getting an error in this case, 

![Screenshot from 2022-01-27 14-44-04](https://user-images.githubusercontent.com/31839712/151328395-49df2e55-1e1f-46da-b2d8-01ba2a4639af.png)

Reply:
Adding `have` solves the writing issue.

Reply:
Already usually comes between the auxiliary (has/have) and the main verb. So better say 'have already installed'. The non-premium version of Grammarly indicates faulty tense too, so let's fix it.

Reply:
Got it clarified. 

`You already.`. is past tense and `You have already..` is present perfect tense.

Past tense: things that have been done already. Present perfect: things that have just been done.

Therefore, since the latter part `you are all set` is in the present tense the former part should be in the present perfect tense rather than the past tense.